pages/logout.tsx

```diff
@@ -1,21 +1,22 @@
 import { useEffect } from 'react'
 import { magic } from 'utils/magic'
-import Router from 'next/router'
+import { useRouter } from 'next/router'
 import Loader from 'components/Loader'
 import { encode as btoa } from 'base-64'
 
 export default function Logout() {
+  const $router = useRouter()
   useEffect(() => {
     const sayGoodbye = async () => {
       if (!magic) return false
       await magic.user.logout()
```

Comment:
The LoginContext/useLogin needs to know that the login state may have changed after this call. If `magic` exposed an event for when login state changes, we could use that in LoginContext/useLogin, but since it doesn't, we should either centralize all auth logic in useLogin and export functions to be used elsewhere, or export functions that allow for modifying the state of useLogin or retriggering `checkLoggedIn`.

Reply:
I ended up adding these two state resets: https://github.com/iron-fish/website-testnet/pull/126/files#diff-ccc3bc2a49ec13caa0caab60e351ddd83198bd11352f1d051b7c70876e284950R22-R23